src/DynamoCoreWpf/Extensions/ViewExtensionManager.cs

```diff
@@ -65,13 +88,38 @@ private IViewExtension RequestLoadViewExtensionHandler(string extensionPath)
             return null;
         }
 
+        private void RequestApplyLayoutSpecHandler(string specJSON)
+        {
+            Log($"an extension requested application of {specJSON} layout spec");
+              try
+                {
+                //try to combine the layout specs.
+                    var customizationService = extensionManager.Service<ILibraryViewCustomization>();
+                //TODO if the layoutspec is empty, we're calling this too early, we can retry after x seconds?
```

Comment:
Have you seen this happening?

Reply:
yes, but only before when I was calling this code from the ExtensionsManager (before view extensions like libraryjs were loaded)